.buildkite/pipelines/pipeline.kibana-release.yaml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - label: ":releasethekraken: Release kibana"
```

Comment:
I'd also clarify it here, that this is a serverless release process, so `Release kibana serverless`